include/libswiftnav/ephemeris.h

```diff
@@ -18,6 +18,9 @@
 #include <libswiftnav/time.h>
 #include <libswiftnav/common.h>
 
+#define SV_TYPE_GLONASS    0
+#define SV_TYPE_GLONASS_M  1
```

Comment:
Do you mean L2C capabilities PR? I don't actually know what is information on GLO SV type is for. So, I'll drop it from the ephemeris. If it indicates G2 capability we'll add it the same way we did for L2C capability in GPS.